src/Rules/StarterWeb/IndividualAuth/Controllers/AccountController.cs

```diff
@@ -43,6 +47,12 @@ public class AccountController : Controller
         [AllowAnonymous]
         public IActionResult Login(string returnUrl = null)
         {
+            // Clear the existing external cookie to ensure a clean login process
+            if (await HttpContext.Authentication.AuthenticateAsync(_externalCookieScheme) != null)
```

Comment:
Might be simpler to just clear the cookie instead of checking?   Authenticate might not be cheap either?